app/src/main/res/layout-land/profile_edit_fragment.xml

```diff
@@ -140,7 +141,7 @@
         android:id="@+id/profile_edit_allow_download_container"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/color_def_white"
+        android:background="@color/component_color_profile_edit_activity_profile_edit_allow_download_container"
```

Comment:
Rename to `component_color_profile_edit_activity_profile_edit_allow_download_container_background_color`

Reply:
done